e2e/playwright-local.config.js

```diff
@@ -15,7 +15,7 @@ const config = {
         env: {
```

Comment:
This entire `env:` block can be removed since it is no longer needed to enable coverage / disable webpack HMR